src/components/UpdateAvatar.js

```diff
@@ -0,0 +1,42 @@
+import Popup from "./Popup.js";
+import api from "./Api.js"; // Import the api instance
+
+class UpdateAvatar extends Popup {
```

Comment:
Needs correction: there should not be such class, you should use PopupWithform for the avatar form